timesketch/ui/graphs/cypher-query.component.html

```diff
@@ -0,0 +1,21 @@
+<div class="card card-top">
+
+  <form class="form-inline" (submit)="onSubmit($event)">
```

Comment:
What are `full post backs`? Are they page reloads? I call `.preventDefault()` manually to prevent page reload since I couldn't find any documentation whether `ngSubmit` and `submit` do that.

Reply:
Yes, full post back = page reload. submit is the standard HTML form event, ngSubmit was especially created for asynchronous scenarios in which you don't want to have a page reload.

This is also the way the Angular documentation recommends: https://angular.io/guide/forms#submit-the-form-with-ngsubmit.